test/src/unittests/spectral/test_logspectrum.py

```diff
@@ -18,88 +18,287 @@
-                            337.18192  ,  332.75058  ,  276.9806   ,  369.5457   ,
-                            423.7845   ,  385.96686  ,  396.63702  ,  417.4205   ,
-                            468.57962  ,  531.24524  ,  657.8269   , 1281.0719   ,
-                            1458.027    ,  698.2347   ,  314.64862  ,  262.41177  ,
-                            250.06857  ,  436.16797  ,  742.8968   ,  696.05994  ,
-                            698.1335   ,  749.1297   ,  626.77167  ,  716.98303  ,
-                            650.1398   , 1007.5442   , 1078.1001   ,  924.02277  ,
-                            954.38226  ,  983.3749   , 1020.8519   , 1221.412    ,
-                            2240.7656   , 2727.331    , 1418.2369   ,  885.09     ,
-                            694.7146   ,  428.4301   ,  396.5824   ,  593.49005  ,
-                            566.84216  ,  518.056    ,  460.924    ,  332.4222   ,
-                            290.82968  ,  236.83658  ,  266.93393  ,  353.3882   ,
-                            270.42496  ,  107.788536 ,   84.7718   ,   91.697174 ,
-                            80.34272  ,   69.842865 ,  130.54848  ,  125.181015 ,
-                            129.66034  ,  138.58502  ,  107.673805 ,  175.68202  ,
-                            182.88759  ,  137.9704   ,   92.93435  ,   71.129166 ,
-                            78.80348  ,  114.65561  ,  125.607994 ,  125.94448  ,
-                            241.2471   ,  270.2437   ,  169.07764  ,  151.46762  ,
-                            174.52895  ,  150.08052  ,  143.80923  ,  181.24687  ,
-                            262.3386   ,  391.69852  ,  237.67671  ,  149.62773  ,
-                            137.92833  ,  178.46875  ,  257.16058  ,  616.3291   ,
-                            660.309    ,  402.2011   ,  349.27042  ,  272.2735   ,
-                            245.1808   ,  255.92816  ,  241.11888  ,  142.76396  ,
-                            95.74622  ,   79.352    ,   71.41603  ,  109.63512  ,
-                            69.190254 ,   38.56327  ,   29.309353 ,   31.107338 ,
-                            44.70423  ,   35.44568  ,   22.666018 ,   16.979908 ,
-                            32.010136 ,   45.12984  ,   34.377556 ,   40.05504  ,
-                            79.10235  ,  127.51099  ,  110.30661  ,   95.92048  ,
-                            86.051285 ,   97.34698  ,   89.825485 ,   62.368706 ,
-                            67.440575 ,   55.217617 ,   40.645    ,   69.08143  ,
-                            58.60874  ,   40.766853 ,   52.087135 ,   75.063484 ])
-
-        audio = MonoLoader(filename = join(testdata.audio_dir, 'recorded/vignesh.wav'),
-                    sampleRate = 44100)()
-
-        w = Windowing(type='hann', normalized=False)
-        spectrum = Spectrum()  # FFT() would return the complex FFT, here we just want the magnitude spectrum
+        expected = array(
```

Comment:
We can re-use the same hardcoded `expected` vector to add more `assertAlmostEqualVector` asserts for different number of octaves.